### DISCUSSION TITLE
recordings namechange automaticly

### DISCUSSION THREAD

**Discussion by JulianBorghuis** [AzuraCast/AzuraCast#4062] [Q&A]
**Using Docker installation method**

Yes

**AzuraCast version**

0.12.2

**Host Operating System**

ubuntu server 16.04

**Describe the bug**

How to change the title of the live dj recordings?
the default template
https://cdn.discordapp.com/attachments/647149735991443467/821042997872427008/unknown.png

if this isnt in azuracast is there an way to change it in ftp of root?
If not, than i wanted to make this an feature request.

**Screenshots**

https://cdn.discordapp.com/attachments/647149735991443467/821042997872427008/unknown.png

**Comment by ivellios1988** [comment]
Update AzuraCast to the newest version, it includes an easy tag editor in its' file manager.
